src/app/components/core/asset-readings/readings-graph/readings-graph.component.css

```diff
@@ -12,11 +12,7 @@
   padding: 15px !important;
 }
 
-::ng-deep .button.is-small {
-  font-size: 0.90rem !important;
-}
-
-::ng-deep .input.is-small {
+::ng-deep .button.small-btn {
```

Comment:
why ng-deep?

Reply:
We are using `time-dropdown` component in readings-graph page as a child component. This css is used to apply css property on `small-btn` class of it's child component `time-dropdown`. So `ng-deep` is required here.

Reply:
I think, this did not break anything. It was breaking before (in develop) which I fixed here. As you can see, I have used custom class `small-btn` which is not being used anywhere. And for safest part, I have also used `:host` before `::ng-host` now.

> why we need to do it? Can't we put the child component css in the css class of the child?

As I told in the reply of another comment, I haven't applied css property in child component css because it may be child component of many other components where we may not need same property.